src/services/bundleAnalysis/useBundleAssetModules.tsx

```diff
@@ -212,5 +221,6 @@ export const useBundleAssetModules = ({
 
         return { modules }
       }),
+    enabled: enabled,
```

Comment:
Do we need a test for this?

Reply:
I guess codecov says we're covered actually lol

Reply:
So the reason why I tested the other one, and not this, is that this is much more straightforward. Whereas the other one we were also checking to see what the repo overview hook was also saying so it was a little more complicated scenario.